analyses/cnv-comparison/util/cnv-comparison-functions.R

```diff
@@ -0,0 +1,200 @@
+# This script defines filtering and plotting functions to be sourced in 
+# `01-cnv-comparison-plotting.R`
+#
+# Chante Bethell for CCDL 2019
+# 
+# Usage:
+# This script is intended to be run via the command line from the top directory 
+# of the repository as follows:
+#
+# Rscript analyses/cnv-comparison/util/cnv-comparison-functions.R
+
+read_in_cnv <- function(input_directory, file_path){
+  # Given the file path of the CNV data, read in the data.
+  #
+  # Arg:
+  #   input_directory: file path of input directory
```

Comment:
Good point, I do see how this can be confusing. I am sure this was better suited for my original, more complicated script, but making it one `file.path` to the data does seem to make the most sense now.